### ISSUE TITLE
Small Issue/error in the code 

### ISSUE THREAD

**Issue by niklasbubeck** [LTH14/mar#47]
Hi @LTH14! 

First of all, thank you for this great work! Love the concept! . To make myself confident with the code I first wanted to train it on tinyImageNet (which only has 200 classes) to get results faster. While doing this I came across the following problem. 

In the file mar/models/mar.py line 59: 
`self.class_emb = nn.Embedding(1000, encoder_embed_dim)`

The given argument class_num of the MAR class has no impact on the class embeddings and instead 1000 classes are hardcoded. 
I thought a pull request might be overkill, but still wanted to make you aware of this. 

Best Niklas

**Comment by LTH14** [comment]
Thanks for pointing out this! Since this code is written and tested only on ImageNet, many places might contain hard-coded class number (1000). The one you mentioned won't cause problem (as the class labels are from 0 to 199), but I have fixed it so that it does not throw an error when num_class>1000.
